en_ID/pages/internship.md

```diff
@@ -14,6 +14,12 @@ Software Engineering Intern (Frontend, JavaScript)  | Sales Development Represen
 Software Engineering Intern (Frontend, Elm)         |
 Software Engineering Intern (Frontend, WordPress)   |
 Technical Writing Intern                            |
+Artificial Intelligence              |
```

Comment:
Hi @Divyansh1011 a PR should be atomic, meaning only contains one solution to a problem at a time. In this case, you try to solve 2 problems that are not favourable.

https://fagnerbrack.com/one-pull-request-one-concern-e84a27dfe9f1?gi=2a3fe2955c5

Reply:
yeh i figured that mistake should i make another branch to solve this ? @empeje

Reply:
also could you guide me how to revert to prv branch so that i can commit the changes to the new branch and create pull request

Reply:
what i do is close this PR and make a new PR?